File: server/dotnet/resources/GameMode/GolemoSDK/Notify.cs
using GTANetworkAPI;
using System;
using System.Collections.Generic;
using System.Text;

namespace GolemoSDK
{
    public enum NotifyType
    {
        Alert,
        Error,
        Success,
        Info,
        Warning
    }
    public enum NotifyPosition
    {
        Top,
        TopLeft,
        TopCenter,
        TopRight,
        Center,
        CenterLeft,
        CenterRight,
        Bottom,
        BottomLeft,
        BottomCenter,
        BottomRight
    }
    public static class Notify
    {
        public static void Send(Player client, NotifyType type, NotifyPosition pos, string msg, int time)
        {
            Trigger.ClientEvent(client, "notify", type, pos, msg, time);
        }
        public static void Error(Player client, string msg, int time)
        {
            Trigger.ClientEvent(client, "notify", NotifyType.Error, NotifyPosition.BottomCenter, msg, time);
        }
        public static void Alert(Player client, string msg, int time)
        {
            Trigger.ClientEvent(client, "notify", NotifyType.Alert, NotifyPosition.BottomCenter, msg, time);
        }
        public static void Info(Player client, string msg, int time)
        {
            Trigger.ClientEvent(client, "notify", NotifyType.Info, NotifyPosition.BottomCenter, msg, time);
        }
        public static void Succ(Player client, string msg, int time)
        {
            Trigger.ClientEvent(client, "notify", NotifyType.Success, NotifyPosition.BottomCenter, msg, time);
        }
        public static void Warn(Player client, string msg, int time)
        {
            Trigger.ClientEvent(client, "notify", NotifyType.Warning, NotifyPosition.BottomCenter, msg, time);
        }
    }
}

File: server/client_packages/browser/assets/js/notify.js
function notify(type, layout, message, time) {
    var types = ['alert', 'error', 'success', 'information', 'warning'];
    var layouts = ['top', 'topLeft', 'topCenter', 'topRight', 'center', 'centerLeft', 'centerRight', 'bottom', 'bottomLeft', 'bottomCenter', 'bottomRight'];
    var icons = ['<i class=""><div class="border"></div><img src="./assets/images/icons/notification/info.png" width="25px">Уведомление! </i></i>', '<i class=""><div class="border"></div><img src="./assets/images/icons/notification/error.png" width="25px">Ошибка! </i>', '<i class=""><div class="border"></div><img src="./assets/images/icons/notification/success.png" width="25px">Успешно! </i></i>', '<i class=""><div class="border"></div><img src="./assets/images/icons/notification/info.png" width="25px">Информация! </i>', '<i class=""><div class="border"></div><img src="./assets/images/icons/notification/warning.png" width="25px" >Предупреждение! </i>']
    message = '<div class="text">'+icons[type]+message+'</div>';
    new Noty({
        type: types[type],
        layout: layouts[layout],
        theme: 'fivestar',
        text: message,
        timeout: time,
        progressBar: true,
        animation: {
            open: 'noty_effects_open',
            close: 'noty_effects_close'
        }
    }).show();
}